### DISCUSSION TITLE
默认的容器标准输出日志无法收集

### DISCUSSION THREAD

**Discussion by Horus-K** [kubesphere/kubesphere#5238] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/48319268/191500099-eb30d7fd-a646-427d-ba39-6877556478bf.png)
![image](https://user-images.githubusercontent.com/48319268/191501228-3b6487e0-d9cf-402d-b2fc-4e085fe77c84.png)

已经开启了日志系统，但在工具里只能搜到一些系统组件的标准输出，我想知道怎么才能搜到

**Comment by rayzhou2017** [comment, marked as answer]
Please go to Chinese forum KubeSphere.com.cn/forum.
